src/routes/web.php

```diff
@@ -29,6 +29,7 @@
     Route::get('/comments/{comment}/edit', [CommentController::class, 'edit'])->name('comments.edit');
     Route::put('/comments/{comment}', [CommentController::class, 'update'])->name('comments.update');
     Route::delete('/comments/{comment}', [CommentController::class, 'destroy'])->name('comments.destroy');
+    Route::resource('profileinfo', ProfileInformationController::class);
```

Comment:
two words? it should be
```suggestion
    Route::resource('profile-info', ProfileInformationController::class);
```

Reply:
ow, Thank you sir didn't know that. Will change it po 🙇🙇🙇